Xcode/pkg-support/resources/CMake/sdl3_mixer-config.cmake

```diff
@@ -0,0 +1,66 @@
+# SDL3_mixer CMake configuration file:
+# This file is meant to be placed in Resources/CMake of a SDL3_mixer framework
+
+# INTERFACE_LINK_OPTIONS needs CMake 3.12
+cmake_minimum_required(VERSION 3.12)
+
+include(FeatureSummary)
+set_package_properties(SDL3_mixer PROPERTIES
+    URL "https://www.libsdl.org/projects/SDL_mixer/"
+    DESCRIPTION "SDL_mixer is a sample multi-channel audio mixer library"
+)
+
+set(SDL3_mixer_FOUND                TRUE)
+
+set(SDL3MIXER_VENDORED              TRUE)
+
+set(SDL3MIXER_CMD                   FALSE)
+
+set(SDL3MIXER_FLAC_LIBFLAC          FALSE)
+set(SDL3MIXER_FLAC_DRFLAC           TRUE)
+
+set(SDL3MIXER_GME                   FALSE)
```

Comment:
I'm assuming here the SDL3_mixer framework won't ship with gme support.

Reply:
I haven't looked at that yet, but I can flip that on if it does.

Reply:
SDL2 version doesn't enable it by default either due to GME being not a very common necessity